samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,351 @@
+
+.. _nrf_cloud_mqtt_multi_services_sample_requirements:
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :sample-yaml-rows:
+
+.. _nrf_cloud_mqtt_multi_services_sample_features:
+
+Features
+********
+
+This sample implements or demonstrates the following features:
+
+* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem <nrfxlib:nrf_modem>` and :ref:`nrf_cloud <lib_nrf_cloud>` libraries.
+* Support for `Firmware Over The Air (FOTA) <nRF Cloud Getting Started FOTA documentation_>`_ via the `nRF Cloud`_ portal.
+* Support for `modem AT Commands <AT Commands Reference Guide_>`_ over UART using the :ref:`AT Host library<lib_at_host>`.
+* Periodic cellular and GNSS location tracking using the :ref:`location library <lib_location>`.
+* Periodic temperature sensor sampling on your `Nordic Thingy:91`_, or fake temperature  measurements on your `Nordic nRF9160 DK`_.
+* Transmission of sensor and GNSS location samples to the `nRF Cloud`_ portal as `nRF Cloud device messages <nRF Cloud Device Messages_>`_.
+* Construction of valid `nRF Cloud device messages <nRF Cloud Device Messages_>`_ using `cJSON`_.
+
+
+.. _nrf_cloud_mqtt_multi_services_sample_structure:
+.. _nrf_cloud_mqtt_multi_services_sample_structure_and_theory:
+.. _nrf_cloud_mqtt_multi_services_sample_structure_and_theory_of_operation:
+.. _nrf_cloud_mqtt_multi_services_sample_theory_of_operation:
+
+Structure and Theory of Operation
+*********************************
+
+This sample is separated into a number of smaller functional units.
+The top level functional unit and entry point for the sample is :file:`src/main.c`.
+This file starts three threads, each with a distinct function:
+
+* Connection Thread (``conn_thread``): Runs the `Connection Management Loop <nrf_cloud_mqtt_multi_services_sample_connection_management_loop_>`_, which maintains a connection to `nRF Cloud`_.
+* Application Thread (``app_thread``): Runs the `Main Application Loop <nrf_cloud_mqtt_multi_services_sample_main_application_loop_>`_, which may add `device messages <nRF Cloud Device Messages_>`_ to the `Device Message Queue <nrf_cloud_mqtt_multi_services_sample_device_message_queue_>`_.
+* Message Thread (``msg_thread``): Processes the `Device Message Queue <nrf_cloud_mqtt_multi_services_sample_device_message_queue_>`_ whenever there is an active connection.
+
+.. _nrf_cloud_mqtt_multi_services_sample_connection_management_loop:
+
+Connection Management Loop
+==========================
+
+The Connection Management Loop performs four primary tasks:
+
+1) Configure and activate the cellular modem.
+#) Start a connection to the LTE Network.
+#) Maintain a connection to `nRF Cloud`_.
+#) Reestablish or retry all lost or failed connections.
+
+Modem setup and the LTE connection are set-and-forget.
+If connection to the LTE is lost, the modem automatically works to reestablish it.
```

Comment:
Can we switch this sentence to third person, since "you" in this case is the user of the sample, and the sample is what actually performs this work? To me, using "you" implies that the sample user will need to perform this task, and that is confusing.

For instance:

`The modem only needs to be set up and connected to LTE once.`

Or:

`This sample only needs to set up the modem and connect to LTE once.`

Reply:
Fine with the third person. My bad trying too hard to make it too active.